dali/python/nvidia/dali/pipeline.py

```diff
@@ -108,14 +108,28 @@ class Pipeline(object):
     unrestricted number of streams is assumed).
 `default_cuda_stream_priority` : int, optional, default = 0
     CUDA stream priority used by DALI. See `cudaStreamCreateWithPriority` in CUDA documentation
-`enable_memory_stats`: bool, optional, default = 1
+`enable_memory_stats`: bool, optional, default = False
     If DALI should print operator output buffer statistics.
-    Usefull for `bytes_per_sample_hint` operator parameter.
+    Useful for `bytes_per_sample_hint` operator parameter.
 `enable_checkpointing`: bool, optional, default = False
     If True, DALI will trace states of the operators. In that case, calling the ``checkpoint``
     method returns serialized state of the pipeline. The same pipeline can be later rebuilt
     with the serialized state passed as the `checkpoint` parameter to resume running
-    from the saved iteration.
+    from the saved iteration::
+
+        @pipeline_def(..., enable_checkpointing=True)
+        def pipeline():
+            ...
+        
+        p = pipeline()
+        p.build()
+        # Run the pipeline...
```

Comment:
nitpick: At first I thought the `Run the pipeline` refers to the line below, where we save the checkpoint. Maybe adding p.run() or even some for loop to make the example more explicit would help disambiguate it?

Or mabe it's just me.

Reply:
Good point, I changed it to explicitly running p.run()